### DISCUSSION TITLE
Excessive TLS handshake errors

### DISCUSSION THREAD

**Discussion by sparkblaze** [minio/minio#16735] [Q&A] [answered]
I'm seeing a large volume of TLS handshake errors across my deployment, but I'm struggling to be able to find a way to actively trigger them in a way that makes troubleshooting obvious or find a reason for it.

Generally,  they're either of the variety `EOF` or `read: connection reset by peer` (another minio node).

Is there any advice or guidance on what I can look towards to find out why I see these?

currently running: minio version RELEASE.2023-01-31T02-24-19Z

**Comment by harshavardhana** [comment, marked as answer]
Upgrade MinIO we have removed these logs
